docs/v4/api.md

```diff
@@ -14,12 +14,12 @@ classDiagram
         +xr.Dataset|xr.DataArray|ux.Dataset|ux.DataArray data
         +parcels.Grid|ux.Grid grid
         +Interpolator interpolator
-        +eval(...)
+        +eval(x,y,z,t,particle)
```

Comment:
I would propose `eval(t, z, y, x, particle)`, following the standardisation of order we just implemented in #1812 

And also add `(..., applyConversion=True)` as default (like `.eval()` has now)?

Reply:
Ah yes, good points.